nuget/framework/nunit.nuspec

```diff
@@ -18,8 +18,7 @@ This package includes the NUnit 3 framework assembly, which is referenced by you
 
 Supported platforms:
 - .NET Framework 3.5+
-- .NET Standard 1.4+
-- .NET Core</description>
```

Comment:
As noted in the description, this isn't really a supported target for the framework. Though I'm not sure if this is desired to leave in for clarity (or searchability ?)

Reply:
Not listing .NET Core is more accurate. Since there's a Dependencies section on the nuget.org pack, this list is duplicate information which usually only makes a difference in a negative way when it gets out of date. @nunit/framework-team  what are your thoughts?

Reply:
If it's duplicate info that has to be kept up to date, I'd favor removal.

Reply:
Also, .NET Core 3.1 is the last version before they drop the word "Core" and go to ".NET 5."

Reply:
Oh, very good points. So as far as current state goes, does this make sense as-is? Or should we remove this part of the description entirely?

Reply:
I think it looks fine. @jnm2 do you agree?

Reply:
Yes.